src/App/component/AddRepositoryDialog.jsx

```diff
@@ -54,16 +54,16 @@ export default function AddRepositoryDialog({open, reloadProjects, handleClose,
               })
           }
         }).catch((e) => {
-        alert(e.response.status)
-        console.error(e)
-      })
+          alert(e.response.status)
```

Comment:
Suggest make sure the `status` field is always exists in `e.response`, otherwise, use a `?` to prevent error.
```suggestion
          alert(e.response?.status)
```
Please also help us change this issues in all place of this PR.